plugins/modules/mso_schema_template_anp_epg.py

```diff
@@ -375,7 +426,15 @@ def main():
         else:
             ops.append(dict(op='add', path=epgs_path + '/-', value=mso.sent))
 
-        mso.existing = mso.proposed
+        if epg_type == 'service':
+            if cloudServiceEpgConfig != {}:
+                cloudServiceEpgConfig.update(dict(deploymentType=deployment_type, serviceType=service_type, accessType=access_type))
```

Comment:
Use a dictionnary to map deployment_type and access_type from lowercase value to correct API value.